blockchain/verifiers/src/test_utils.rs

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Test helpers.
+
+use crate::{
+    metadata_verifiers::{avr::AvrVerifier, message_signing_key::MessageSigningKeyVerifier},
+    AvrConfig, AvrConfigRecord, MetadataVerifier,
+};
+use mc_attest_core::{VerificationReport, VerificationSignature};
+use mc_blockchain_test_utils::{make_block_id, make_block_metadata_contents};
+use mc_blockchain_types::{BlockIndex, BlockMetadata};
+use mc_common::ResponderId;
+use mc_crypto_keys::{Ed25519Pair, Ed25519Public};
+use mc_util_from_random::FromRandom;
+use mc_util_test_helper::{RngType as FixedRng, SeedableRng};
+use std::str::FromStr;
+
+pub fn make_key(seed: u64) -> Ed25519Public {
+    *make_metadata(seed).node_key()
+}
+
+pub fn make_metadata(seed: u64) -> BlockMetadata {
+    let seed_bytes = seed.to_be_bytes();
+    let mut seed = [0u8; 32];
+    seed[0..seed_bytes.len()].copy_from_slice(&seed_bytes);
+    let mut rng = FixedRng::from_seed(seed);
```

Comment:
I recently discovered [`SeedableRng::seed_from_u64`](https://docs.rs/rand/latest/rand/trait.SeedableRng.html#method.seed_from_u64); let's use that here.
```suggestion
    let mut rng = FixedRng::seed_from_u64(seed);
```